pageserver/src/page_service.rs

```diff
@@ -944,7 +1014,10 @@ impl PageServerHandler {
                         })
                     }
                 },
-                Ok(response_msg) => response_msg,
+                Ok((response_msg, timer)) => {
+                    timers.push(timer);
```

Comment:
IMO this deserves a comment. We are extending the lifetime of the timers until after the flush.

Reply:
5d778eecd1aee14165c50365d34d35b1a93ba99a